homeassistant/components/zwave_js/climate.py

```diff
@@ -40,14 +38,46 @@
 from homeassistant.helpers.entity_platform import AddEntitiesCallback
 from homeassistant.util.unit_conversion import TemperatureConverter
 
-from .const import DATA_CLIENT, DOMAIN
+from .const import DATA_CLIENT, DOMAIN, LOGGER
 from .discovery import ZwaveDiscoveryInfo
 from .discovery_data_template import DynamicCurrentTempClimateDataTemplate
 from .entity import ZWaveBaseEntity
 from .helpers import get_value_of_zwave_value
 
 PARALLEL_UPDATES = 0
 
+THERMOSTAT_MODES = [
+    ThermostatMode.OFF,
+    ThermostatMode.HEAT,
+    ThermostatMode.COOL,
+    ThermostatMode.AUTO,
+    ThermostatMode.AUTO_CHANGE_OVER,
+    ThermostatMode.FAN,
+    ThermostatMode.DRY,
+]
+
+THERMOSTAT_MODE_SETPOINT_MAP: dict[int, list[ThermostatSetpointType]] = {
```

Comment:
Is this map something specific for Home Assistant or a general relationship for the Z-Wave specification or Z-Wave JS?

Reply:
@raman325 I'll pass this question on to you since you know better.

Reply:
I guess this can be applied elsewhere as well since it's just mapping zwave values/property keys to each other. We can revert in the repairs issue PR

Reply:
Reverted in https://github.com/home-assistant/core/pull/95997